justfile

```
@@ -208,7 +208,7 @@ clean-tools:
 
 # Removes all asset data
 remove-assets:
-    find $FLUTTER_EXAMPLE_WORKSPACE/assets/*_v* -type f ! -name .gitkeep -exec rm '{}' \;
+    find $FLUTTER_EXAMPLE_WORKSPACE/assets/*_v* -type f ! -name .gitkeep ! -name *mocked.onnx -exec rm '{}' \;
```

:wrench: should be `'*mocked.onnx'` or else it will fail if there is a file ending with `mocked.onnx` in the working dir

```suggestion
    find $FLUTTER_EXAMPLE_WORKSPACE/assets/*_v* -type f ! -name .gitkeep ! -name '*mocked.onnx' -exec rm '{}' \;
```

:question:  Maybe  `'*-mocked.onnx'` be a bit more robust but I'm not sure.
top 

```suggestion
    find $FLUTTER_EXAMPLE_WORKSPACE/assets/*_v* -type f ! -name .gitkeep ! -name '*-mocked.onnx' -exec rm '{}' \;
```